src/app/api/v1/onerep-events/route.ts

```diff
@@ -32,49 +32,35 @@ interface OnerepWebhookRequest {
 export async function POST(req: NextRequest) {
   let finalBuffer: Buffer;
   try {
-    if (!process.env.ONEREP_WEBHOOK_SECRET) {
-      throw new Error("env var ONEREP_WEBHOOK_SECRET must be set");
-    }
-
     const buffers: Buffer[] = [];
     // @ts-ignore FIXME Type error: Type 'ReadableStream<Uint8Array>' must have a '[Symbol.asyncIterator]()' method that returns an async iterator.
     for await (const data of req.body) {
       buffers.push(data);
     }
     finalBuffer = Buffer.concat(buffers);
 
-    const actualSignature = req.headers.get("signature");
-
-    const expectedSignature = crypto
-      .createHmac("sha256", process.env.ONEREP_WEBHOOK_SECRET)
-      .update(finalBuffer)
-      .digest("hex");
-
-    if (actualSignature !== expectedSignature) {
-      throw new Error("Webhook signature invalid");
+    if (!contentSignatureValid(finalBuffer, req.headers.get("signature"))) {
+      logger.error("onerep_webhook_signature_invalid");
+      return NextResponse.json({ success: false }, { status: 401 });
     }
   } catch (ex) {
-    logger.error(ex);
+    logger.error("onerep_webhook_auth_exception", ex);
     captureException(ex);
 
-    return NextResponse.json({ success: false }, { status: 401 });
+    return NextResponse.json({ success: false }, { status: 500 });
   }
-
   try {
     const result: OnerepWebhookRequest = JSON.parse(finalBuffer.toString());
     logger.debug("OneRep Webhook Request received:", result);
 
     if (result.type !== "scan.completed") {
-      logger.debug("Unexpected OneRep webhook type received:", result.type);
-      return;
+      logger.info("ignored_onerep_webhook_type", result.type);
```

Comment:
Any differences between `logger.debug()` and `logger.info()`?

Reply:
Currently we don't log `debug` level messages in GCP, just for local development. Also FWIW the reason I'm changing the message name is because it's easier to handle these in BigQuery if they don't have spaces.